### PR TITLE
[es] fix broken ref

### PR DESCRIPTION
**Pull request by Arhell**
en PR https://github.com/kubernetes/website/pull/30573/files
/assign @electrocucaracha
/assign @raelga

### CODE REVIEW

**Comment by electrocucaracha** [comment]
@Arhell this fix doesn't apply here given the line 406 uses `{#nodeport}` as identifier. So you need to modify the reference as well to `{#type-nodeport}` to keep it working

**Comment by Arhell** [comment]
> @Arhell this fix doesn't apply here given the line 406 uses `{#nodeport}` as identifier. So you need to modify the reference as well to `{#type-nodeport}` to keep it working

Done

File: content/es/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -393,7 +393,7 @@ Los `ServiceTypes` de Kubernetes permiten especificar qué tipo de Service quier
 Los valores `Type` y sus comportamientos son:
 
 - `ClusterIP`: Expone el Service en una dirección IP interna del clúster. Al escoger este valor el Service solo es alcanzable desde el clúster. Este es el `ServiceType` por defecto.
-- [`NodePort`](#nodeport): Expone el Service en cada IP del nodo en un puerto estático (el `NodePort`). Automáticamente se crea un Service `ClusterIP`, al cual enruta el `NodePort`del Service. Podrás alcanzar el Service `NodePort` desde fuera del clúster, haciendo una petición a `<NodeIP>:<NodePort>`.
+- [`NodePort`](#tipe-nodeport): Expone el Service en cada IP del nodo en un puerto estático (el `NodePort`). Automáticamente se crea un Service `ClusterIP`, al cual enruta el `NodePort`del Service. Podrás alcanzar el Service `NodePort` desde fuera del clúster, haciendo una petición a `<NodeIP>:<NodePort>`.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/services-networking/service.md:396]:
```suggestion
- [`NodePort`](#tipo-nodeport): Expone el Service en cada IP del nodo en un puerto estático (el `NodePort`). Automáticamente se crea un Service `ClusterIP`, al cual enruta el `NodePort`del Service. Podrás alcanzar el Service `NodePort` desde fuera del clúster, haciendo una petición a `<NodeIP>:<NodePort>`.
```

File: content/es/docs/concepts/services-networking/service.md [hunk 2]
```diff
@@ -403,7 +403,7 @@ Los valores `Type` y sus comportamientos son:
 
 También puedes usar un [Ingress](/docs/concepts/services-networking/ingress/) para exponer tu Service. Ingress no es un tipo de Service, pero actúa como el punto de entrada de tu clúster. Te permite consolidar tus reglas de enrutamiento en un único recurso, ya que puede exponer múltiples servicios bajo la misma dirección IP.
 
-### Tipo NodePort {#nodeport}
+### Tipo NodePort {#tipe-nodeport}
```
Comment (electrocucaracha) [inline content/es/docs/concepts/services-networking/service.md:406]:
```suggestion
### Tipo NodePort {#tipo-nodeport}
```

File: content/es/docs/concepts/services-networking/service.md (hunk 1 above)
Comment (Arhell) [reply content/es/docs/concepts/services-networking/service.md]: Done thanks

File: content/es/docs/concepts/services-networking/service.md (hunk 2 above)
Comment (Arhell) [reply content/es/docs/concepts/services-networking/service.md]: Done thanks

**Pull request merged**